File: Course_01/C++ Language/EstruturaRepetitiva/soma_impares.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int x, y, temp, primeiroImpar, soma(0), i;

    cout << "Digite dois numeros: " << endl;
    cin >> x >> y;

    if (x > y) {
        temp = x;
        x = y;
        y = temp;
    }

    if (x % 2 == 0) {
        primeiroImpar = x + 1;
    }
    else if (x % 2 != 0) {
        primeiroImpar = x + 2;
    }

    for (i = primeiroImpar; i < y; i+=2) {
        soma += i;
    }
    cout << "SOMA DOS IMPARES = " << soma << endl;
    return 0;
}

File: Course_01/Python/EstruturasRepetitivas/soma_impares.py
print("Digite dois números:")
x = int(input())
y = int(input())
soma = 0

if x > y:
    temp = y
    y = x
    x = temp

if x % 2 == 0:
    x += 1
else:
    x += 2
for i in range (x, y, 2):
    soma += i
print(f"SOMA DOS IMPARES = {soma}")
